docs/troubleshooting/README.md

```diff
@@ -0,0 +1,14 @@
+# thundernetes Troubleshooting Guide
+
+This is a public repository for all of thundernetes Troublesooting guides, and is intended to provide a central location for community driven troubleshooting content. 
```

Comment:
nit: Troubleshooting

Reply:
Spell check! Opps! Thanks for the catch!